middleware.js

```diff
@@ -83,10 +84,7 @@ module.exports = function(options) {
   // Enable debug output
   var debug = options.debug;
 
-  var indentedSyntax = options.indentedSyntax || null;
-  var sassExtension = (indentedSyntax === true) ? '.sass' : '.scss';
-
-  var sourceMap = options.sourceMap || null;
```

Comment:
Revert change on line 89, it is used on [line 175](https://github.com/teodragovic/node-sass-middleware/blob/master/middleware.js#L175).